awx/ui_next/src/components/FormActionGroup/FormActionGroup.jsx

```diff
@@ -24,16 +18,6 @@ const FormActionGroup = ({
         >
           {i18n._(t`Save`)}
         </Button>
-        {onRevert && (
-          <Button
-            aria-label={i18n._(t`Revert`)}
-            variant="secondary"
-            type="button"
-            onClick={onRevert}
-          >
-            {i18n._(t`Revert`)}
-          </Button>
-        )}
```

Comment:
Since the ability to revert is a settings-only feature, I removed the onRevert button from the top-level FormActionGroup component and created a custom  action button group for Settings called RevertFormActionGroup. The RevertFormActionGroup component allows the form to pass in additional action buttons as `{ children }` and keeps the top-level FormActionGroup generic.